### ISSUE TITLE
Error: [ng:cpws] Can't copy! Making copies of Window or Scope instances is not supported.

### ISSUE THREAD

**Issue by icxDave** [json-schema-form/angular-schema-form#849]
Error: [ng:cpws] Can't copy! Making copies of Window or Scope instances is not supported.
When upgrading to Angular 1.6.0+, upon 
`$scope.$broadcast('schemaFormValidate');`

there is an error with angular 1.6.0+ that will prevent the $$scope from when Angular copies the element/ng-model:

Stack Trace:
<img width="617" alt="screen shot 2017-03-01 at 7 09 12 pm" src="https://cloud.githubusercontent.com/assets/14094547/23511010/be78eefa-ff29-11e6-8983-980557e558c8.png">

Angular is copying the element based on the stack track:
<img width="394" alt="screen shot 2017-03-01 at 7 11 07 pm" src="https://cloud.githubusercontent.com/assets/14094547/23511009/be778cf4-ff29-11e6-83ee-df12f9641bc4.png">

Here is a snippet of what the object looks like - which includes $$scope.
<img width="446" alt="screen shot 2017-03-01 at 9 34 28 pm" src="https://cloud.githubusercontent.com/assets/14094547/23511011/be7bba68-ff29-11e6-93ce-8ccd8700a90f.png">

I've tried modifying the $$scope that's to be copied but it breaks when it needs the $watchers or $evalAsSync in later operations.

WIth 1.6.0 Angular - it's checking for the below to ensure we aren't copying Window or $scope.
````
function isWindow(obj) {
  return obj && obj.window === obj;
}

function isScope(obj) {
  return obj && obj.$evalAsync && obj.$watch;
}
````

I've tried searching for someone that has run into this issue but haven't seen any posts yet for it.

@json-schema-form/angular-schema-form-lead

**Comment by Anthropic** [comment]
@icxDave thanks for letting me know, it looks like it may be a bug in 1.6 to be fixed in 1.6.1 based on reading this:
https://github.com/angular/angular.js/issues/15479

Could you try their solution and let me know if it fixes this or not?

**Comment by Anthropic** [comment]
@icxDave any luck with the proposed solution?

I'm going to close this unless/until someone confirms this isn't an Angular bug.
